docs/02_database/05_database-interface/01_entitydb.md

```diff
@@ -21,7 +21,55 @@ entity. It supports read operations for views and tables and write operations fo
 
 The entity db differs from the generated repositories in that it can handle any table and most view entities. It differs from `RxDb` in that all operations are type-safe.
 
-The entity db is available in the kotlin Event Handler. It can be injected in Kotlin using `AsyncEntityDb` and in Java using `RxEntityDb`.
+The entity db is available in the kotlin Event Handler. 
+
+
+It can be injected in Kotlin using `AsyncEntityDb` and in Java using `RxEntityDb`.
+
+There are three versions available of the entity db:
+* `AsyncEntityDb` - this API support kotlin coroutines 
+* `SyncEntityDb` - this API is blocking 
+* `RxEntityDb` - this API supports RxJava
+
+:::important
+
+The `SyncEntityDb` API was introduced in version 7.0 of the Genesis platform.
+
+:::
+
+## Which version to use?
+
+The version to use depends on the context. It depends on whether you are writing code in kotlin or java, and also 
+whether there is a need to write asynchronous (async) or synchronous (blocking) code. 
```

Comment:
maybe a 'preference' rather than a `need` ?

Reply:
it depends on the context, sometimes there is a need, for example in gpal you need to use the async api. Sometimes there is a preference.